src/vivarium_ihme_obesity_washout/components/observer.py

```diff
@@ -0,0 +1,106 @@
+import pandas as pd
+
+from vivarium_public_health.disease import DiseaseModel, RiskAttributableDisease
+from vivarium_public_health.metrics.utilities import get_age_bins, get_person_time, get_deaths, get_years_of_life_lost
+
+
+class BMIObserver:
+    configuration_defaults = {
+        'metrics': {
+            'bmi': {
+                'by_age': False,
+                'by_year': False,
+                'by_sex': False,
+                'categories':  [15, 18.5, 25, 30, 35, 40, 60],  # right edges (exclusive) of BMI categories to bin by
+                'washout': {
+                    'program_start': {
+                        'year': 2000,
+                        'month': 7,
+                        'day': 9,
+                    },
+                    'duration': 5  # years
+                }
+            }
+        }
+    }
+
+    @property
+    def name(self):
+        return 'bmi_observer'
+
+    def setup(self, builder):
+        self.config = builder.configuration.metrics.bmi
+        self.clock = builder.time.clock()
+        self.step_size = builder.time.step_size()
+        self.start_time = self.clock()
+
+        program_start = pd.Timestamp(month=self.config.washout.program_start.month,
```

Comment:
program_start = pd.Timestamp(**self.config.washout.program_start.to_dict())

Reply:
Changed! I always forget that config trees have that handy to_dict